packages/jest-reporters/src/__tests__/get_watermarks.test.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
can the test be in ts?

Reply:
Absolutely, but you need to use the `makeGlobalConfig` helper to make typescript happy. I did it locally but then suddenly had problems getting the tests to run :(

I did this using wsl because or line endings. Because most windows users use autocrlf cloning jest results in code that won't satisfy prettier and using vscode with prettier integration is impossible because you drown in errors. Since autocrlf is a global setting and I need it for other repositories my solution was to clone jest and edit it using wsl.

Reply:
Weird prettier cares - we haven't configured it so it should just ensure it's consistent... We could add a `.gitattributes` to this repo file with the correct endings perhaps?
You should also be able to set config just for this repo without messing with your global defaults.

https://prettier.io/docs/en/options.html#end-of-line

Reply:
This is what's causing my problems: https://github.com/facebook/jest/pull/8809/files#diff-1e70daafb475c0ce3fef7d2728279182R13

Reply:
Also the errors I get when running coverage_reporter.test.js is this: 
`ENOENT: no such file or directory, lstat 'local path on my machine/jest/node_modules/callsites'` I don't know if you've seen it before?